spec/core/ics-002-client-semantics/README.md

```diff
@@ -552,15 +552,12 @@ logical correctness.
 Calling `createClient` with the client type and initial consensus state creates a new client.
 
 ```typescript
-function createClient(
-  clientType: ClientType,
-  consensusState: ConsensusState) {
-    // implementations may define a identifier generation function
-    identifier = generateClientIdentifier()
-    abortTransactionUnless(provableStore.get(clientStatePath(identifier)) === null)
-    abortSystemUnless(provableStore.get(clientTypePath(identifier)) === null)
-    clientType.initialise(identifier, consensusState)
-    provableStore.set(clientTypePath(identifier), clientType)
+function createClient(consensusState: ConsensfusState) {
+  // implementations may define a identifier generation function
+  identifier = generateClientIdentifier()
+  abortTransactionUnless(provableStore.get(clientStatePath(identifier)) === null)
+  clientState = initialise(identifier, consensusState)
```

Comment:
@AdityaSripal I did this differently to what we discussed: I thought leaving the `initialise` function as it was (i.e. creating the client state) was easier and involved fewer changes. Let me know what you think.

Reply:
Diverges from the implementation:

https://github.com/cosmos/ibc-go/blob/main/modules/core/02-client/keeper/client.go#L17

Reply:
Yes, it does diverge... Ok, I will make it comply with implementation, but I will need to do some changes in ICS 06, 07 and 09 as well.